src/components/SwatchPaletteWidget/SwatchPalette.js

```diff
@@ -1,554 +1,570 @@
+      hex: coolGray[20],
+      pms: '650',
       cmyk: 'c13 m9 y10 k0',
     },
     '30': {
-      hex: '#B9BFC7',
-      pms: 'Cool Gray 3',
+      hex: coolGray[30],
+      pms: '537',
       cmyk: 'c25 m20 y21 k0',
     },
     '40': {
-      hex: '#9FA5AD',
-      pms: 'Cool Gray 4',
+      hex: coolGray[40],
+      pms: '2162',
       cmyk: 'c38 m30 y31 k0',
     },
     '50': {
-      hex: '#868D95',
-      pms: 'Cool Gray 6',
+      hex: coolGray[50],
+      pms: '2163',
       cmyk: 'c47 m39 y39 k4',
     },
     '60': {
-      hex: '#697077',
-      pms: 'Cool Gray 7',
+      hex: coolGray[60],
+      pms: '2164',
       cmyk: 'c57 m48 y48 k15',
     },
     '70': {
-      hex: '#50565B',
-      pms: 'Cool Gray 8',
+      hex: coolGray[70],
+      pms: '2165',
       cmyk: 'c63 m55 y55 k30',
     },
     '80': {
-      hex: '#373D42',
-      pms: 'Cool Gray 9',
+      hex: coolGray[80],
+      pms: '2166',
       cmyk: 'c68 m61 y60 k49',
     },
     '90': {
-      hex: '#242a2e',
-      pms: 'Cool Gray 10',
+      hex: coolGray[90],
+      pms: '7545',
       cmyk: 'c71 m64 y64 k68',
     },
     '100': {
-      hex: '#13171A',
-      pms: 'Cool Gray 11',
+      hex: coolGray[100],
+      pms: '7546',
       cmyk: 'c73 m67 y65 k79',
     },
   },
   cyan: {
     '10': {
-      hex: '#E3F6FF',
+      hex: cyan[10],
       pms: '9421',
       cmyk: 'c9 m0 y0 k0',
     },
     '20': {
-      hex: '#B3E6FF',
+      hex: cyan[20],
       pms: '2905',
       cmyk: 'c26 m0 y0 k0',
     },
     '30': {
-      hex: '#6CCAFF',
+      hex: cyan[30],
       pms: '2190',
       cmyk: 'c48 m5 y0 k0',
     },
     '40': {
-      hex: '#30B0FF',
+      hex: cyan[40],
       pms: '299',
       cmyk: 'c62 m18 y0 k0',
     },
     '50': {
-      hex: '#1191e6',
+      hex: cyan[50],
       pms: '2192',
       cmyk: 'c72 m32 y0 k0',
     },
     '60': {
-      hex: '#0072C3',
+      hex: cyan[60],
       pms: '2194',
       cmyk: 'c86 m52 y0 k0',
     },
     '70': {
-      hex: '#0058A1',
+      hex: cyan[70],
       pms: '2195',
       cmyk: 'c96 m71 y5 k0',
     },
     '80': {
-      hex: '#003D73',
+      hex: cyan[80],
       pms: '300',
       cmyk: 'c100 m83 y29 k14',
     },
     '90': {
-      hex: '#002b50',
+      hex: cyan[90],
       pms: '301',
       cmyk: 'c100 m84 y40 k39',
     },
     '100': {
-      hex: '#07192B',
+      hex: cyan[100],
       pms: '302',
       cmyk: 'c91 m78 y54 k68',
     },
   },
   gray: {
     '10': {
-      hex: '#F3F3F3',
-      pms: '656',
+      hex: gray[10],
+      pms: 'Cool Gray 1',
       cmyk: 'c4 m2 y1 k0',
     },
     '20': {
-      hex: '#DCDCDC',
-      pms: '650',
+      hex: gray[20],
+      pms: 'Cool Gray 2',
       cmyk: 'c15 m10 y7 k0',
     },
     '30': {
-      hex: '#BEBEBE',
-      pms: '537',
+      hex: gray[30],
+      pms: 'Cool Gray 3',
       cmyk: 'c27 m18 y15 k0',
     },
     '40': {
-      hex: '#A4A4A4',
-      pms: '2162',
+      hex: gray[40],
+      pms: 'Cool Gray 4',
```

Comment:
It looks like some of these got swapped or something? Is it supposed to be an integer?